OpenBullet/Pages/Main/Configs/ConfigManager.xaml.cs

```diff
@@ -148,9 +149,16 @@ public void SaveConfig()
             Current.Config.Settings.LastModified = DateTime.Now;
             Current.Config.Settings.Version = Globals.obVersion;
             Globals.LogInfo(Components.ConfigManager, "Converted the unbinded observables and set the Last Modified date");
-            
+
+            if (File.Exists(Current.Path) && ((Current.Config.Settings.Name + ".loli") != Current.Path.Replace(Globals.configFolder + "\\", "")))
+            {
+                string newpath = Globals.configFolder + "\\" + (Current.Category == "Default" ? "" : (Current.Category + "\\")) + Current.Config.Settings.Name + ".loli";
```

Comment:
You can combine paths with the `Path.Combine()` method. Do not reinvent the wheel mate!
e.g.
```csharp
string newPath = Path.Combine(Globals.configFolder, Current.Category, Current.Config.Settings.Name + ".loli");
```

Reply:
Another way to do this (much easier and that automatically accounts for all the subfolder or no subfolder) is this one:
```csharp
string newPath = Path.Combine(Path.GetDirectoryName(Current.Path), Current.Config.Settings.Name);
```